api/c/indigo/src/indigo_loaders.h

```diff
@@ -149,7 +151,7 @@ class IndigoJSONMolecule : public IndigoObject
 protected:
     Molecule _mol;
     rapidjson::Value& _node;
-    rapidjson::Value& _rgroups;
+    std::list<std::pair<int, std::reference_wrapper<Value>>>& _rgroups;
```

Comment:
If `Value` inside `lengthy `reference_wrapper<Value>` is `rapidjson::Value`, then it must be typed so: `...reference_wrapper<rapidjson::Value>...`

One more point to create an alias.

Reply:
Fixed